src/ert/_c_wrappers/config/config_parser.py

```diff
@@ -13,10 +14,35 @@ class ConfigWarning(UserWarning):
     pass
 
 
+@dataclass
+class ErrorInfo:
+    line: int
+    end_line: int
+    column: int
+    end_column: int
+    message: Optional[str]
+    data: Optional[dict]
+    original_error: Optional[Exception]
```

Comment:
You could make a classmethod `from_token` that returns None if not a token and The location info from the token otherwise.

Reply:
I think you might want this instead:
```python
@dataclass
class LocationInfo:
   line: int
   column: int
   file: str
```

Reply:
On second thought, I think we can just not introduce a class for this at all, and use FileContextToken directly instead? 
FCToken does exactly the same as it does, so it would be introducing a differently named class containing exactly the same info